samples/Catalog/Catalog.android.js

```diff
@@ -103,6 +103,22 @@ var examples = [
       component.props.navigation.navigate("PdfView");
     }
   },
+  {
+    name: "PDF View Component Listeners",
+    description:
+      "Show how to use the listeners exposed by PSPDFKitView component.",
+    action: component => {
+      component.props.navigation.navigate("PdfViewListenersScreen");
+    }
+  },
+  {
+    name: "PDF View Instant JSON",
+    description:
+      "Shows how to get and add new annotations using instant json.",
```

Comment:
Instant JSON needs to be capitalized: "Shows how to get and add new annotations using Instant JSON."